gamification-crowdin-services/src/main/java/io/meeds/gamification/crowdin/services/WebhookService.java

```diff
@@ -169,6 +158,22 @@ public List<WebHook> getWebhooks(int offset, int limit, boolean forceUpdate) {
         return getWebhooks(offset, limit);
     }
 
+    public List<RemoteDirectory> getProjectDirectories(
+            long remoteProjectId, String currentUser, int offset, int limit
+    ) throws IllegalAccessException, ObjectNotFoundException {
+
+        if (!Utils.isRewardingManager(currentUser)) {
```

Comment:
why checking this ?

Reply:
> why checking this ?

It is because so far, the directories are fetched from Crowdin when the admin is creating an action.